phpstan-baseline.php

```diff
@@ -21,6 +21,16 @@
 	'count' => 13,
 	'path' => __DIR__ . '/system/Autoloader/FileLocator.php',
 ];
+$ignoreErrors[] = [
+	'message' => '#^Only booleans are allowed in a ternary operator condition, int\\|string given\\.$#',
+	'count' => 1,
+	'path' => __DIR__ . '/system/Autoloader/FileLocator.php',
+];
+$ignoreErrors[] = [
+	'message' => '#^Only booleans are allowed in an if condition, string given\\.$#',
```

Comment:
Rector should can handle it:

https://getrector.com/demo/f64fdd76-8ad2-4af6-b57d-d8aa1ef366e8

with ` BooleanInIfConditionRuleFixerRector `

Reply:
I created PR:

- https://github.com/codeigniter4/CodeIgniter4/pull/7951

for it.